pydicom/tests/test_encoders.py

```diff
@@ -308,7 +277,7 @@ def test_iter_encode_invalid_type_raises(self):
     # Passing bytes
     def test_bytes(self):
         """Test encoding bytes"""
-        assert 32768 == len(self.bytes)
+        assert len(self.bytes) == 32768
```

Comment:
The encoded lengths changed because the temporary solution was encoding the entire pixel array as 1D array (i.e. one giant row), whereas now it gets encoded properly on a row-by-row basis which adds some overhead.